vmdb/app/models/vm_or_template.rb

```diff
@@ -667,6 +667,19 @@ def disconnect_inv
     end
 
     self.disconnect_host
+
+    disconnect_stack if respond_to?(:orchestration_stack)
+  end
+
+  def disconnect_stack(stack = nil)
+    return unless orchestration_stack
+    return if stack && stack != orchestration_stack
+
+    log_text = " from stack [#{orchestration_stack.name}] id [#{orchestration_stack.id}]"
+    $log.info "MIQ(#{self.class.name}#disconnect_stack) Disconnecting Vm [#{name}] id [#{id}]#{log_text}"
```

Comment:
Please update to the new `_log` call that @matthewd created in another PR.  I believe it should be something like:

`_log.info "Disconnecting Vm [#{name}] id [#{id}]#{log_text}"`